src/components/Todo.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from "react";
+
+class Todo extends Component {
+  constructor() {
+    super();
+    this.state = {};
+  }
+  render() {
+    return (
+      <li
+        style={{
+          textDecoration: this.props.task.completed ? "line-through" : "",
+        }}
+        onClick={() => this.props.toggleCompleted(this.props.key)}
```

Comment:
incearca sa nu folosesti `this.props.key`, noi la lectie am vorbit ca `key` este ceva numai pentru React. Noi nu ar trebui sa avem acces la el

Reply:
Am schimbat id in lok de key. Dar oare cum sa fac legatura cu functia removeCompleted() astfel incat ea sa primeasca proprietatea completed:true??